src/Elasticsearch/Bundle/Command/CreateIndexCommand.php

```diff
@@ -41,6 +45,8 @@ protected function configure(): void
             ->setDefinition([
                 new InputOption('re-create-indexes', '', InputOption::VALUE_REQUIRED, 'Re-create existing indexes (delete exists data)', false),
                 new InputOption('select', '', InputOption::VALUE_REQUIRED, 'Select indexes for create', false),
+                new InputOption('byName', '', InputOption::VALUE_REQUIRED, 'Index name without prefix for delete'),
+                new InputOption('byClassName', '', InputOption::VALUE_REQUIRED, 'Index class name for delete'),
```

Comment:
Tady má být asi Index class name for index. Ne delete, zkopírováno z druhého commandu ?

Reply:
Jj, v tom popisu jsem to zapomel upravit... klasika. Diky, upraveno.